assets/js/script.js

```diff
@@ -1,12 +1,66 @@
-document.getElementById('add-button').addEventListener('click', function() {
+document.getElementById('add-button').addEventListener('click', function () {
     const input = document.getElementById('item-input');
     const listContainer = document.getElementById('list-container');
     const listItem = document.createElement('li');
 
     if (input.value.trim() !== '') {
         listItem.innerHTML = `${input.value} <button class="delete is-small"> </button>`;
         listContainer.appendChild(listItem);
-        input.value = ''; 
+        input.value = '';
 
     }
 });
+
+// Modal JS Functionality Implementation Below:
+
+// Clicking the 'x' button in the top right: 
+document.addEventListener('DOMContentLoaded', function () {
+    const modal = document.querySelector('.modal');
+    const modalButton = document.querySelector('.button');
+    const closeButton = modal.querySelector('.delete');
+
+    // Fx to open the modal
+    function openModal() {
+        modal.classList.add('is-active');
+    }
+
+    // Fx to close the modal
+    function closeModal() {
+        modal.classList.remove('is-active');
+    }
+
+    // Event listener for the button to open the modal
+    modalButton.addEventListener('click', closeModal);
+
+    // Event listener for the close button within the modal
+    closeButton.addEventListener('click', closeModal);
+});
+
+
+
+// Clicking the 'Let's Get Started!' button:
+document.addEventListener('DOMContentLoaded', function () {
+    const modal = document.querySelector('.modal');
+    const modalButton = document.querySelector('.button');
+    const closeButton = modal.querySelector('.delete');
+    const startButton = modal.querySelector('.modal-card-foot .button');
+
+    // Fx to open the modal
+    function openModal() {
+        modal.classList.add('is-active');
+    }
```

Comment:
This function looks like a duplicate of line 22, but maybe that was done on purpose. When I read this, it seems like a user will be able to open the modal even after clicking the "x" or "Let's Get Started" button. Is that what this does?

Reply:
That is incorrect. I added this line of code underneath the "Let's Get Started!" Button section, as well as underneath the code for the "x" in the upper right hand corner of the modal. It is written this way because it activates the modal to immediately be open and linked to the Event Listener of clicking the 'Lets Get Started' button. Without this line of code, the 'Let's Get Started!' button is inactive and does not work.